.jvmopts

```diff
@@ -0,0 +1 @@
+-XX:-TieredCompilation
```

Comment:
I was thinking more along the lines of

```
-XX:MaxPermSize=256m
-Xss2m
-Xms725m
-Xmx725m
```

i.e. specifying your exact requirements.